code/package.json

```diff
@@ -251,6 +251,7 @@
     "process": "^0.11.10",
     "raf": "^3.4.1",
     "react": "^16.14.0",
+    "safe-stable-stringify": "^2.4.3",
```

Comment:
@valentinpalkovic when you have a moment can you remove it from this package.json file? Thanks in advance